extensions/default/src/customizations/studyBrowserCustomization.ts

```diff
@@ -66,32 +66,24 @@ export default {
   'studyBrowser.studyMode': 'all',
   'studyBrowser.thumbnailDoubleClickCallback': {
     callbacks: [
-      ({ activeViewportId, servicesManager, commandsManager, isHangingProtocolLayout }) =>
+      ({ activeViewportId, commandsManager, servicesManager }) =>
         async displaySetInstanceUID => {
-          const { hangingProtocolService, uiNotificationService } = servicesManager.services;
-          let updatedViewports = [];
-          const viewportId = activeViewportId;
-
           try {
-            updatedViewports = hangingProtocolService.getViewportsRequireUpdate(
-              viewportId,
-              displaySetInstanceUID,
-              isHangingProtocolLayout
-            );
+            commandsManager.run('loadSegmentationDisplaySetsForViewport', {
```

Comment:
Hmmm i don't follow this logic, why we are saying the default is loadSegmentation?

Reply:
Looking at the implementation of loadSegementationDisplaySetsForViewport it seems exactly the same. Am i missing something? still a bit new to this repo

Reply:
there is nothing segmentation wise for the study browser